src/plugins/data/public/ui/dataset_selector/dataset_selector.tsx

```diff
@@ -99,22 +99,41 @@ export const DatasetSelector = ({
         prepend: <EuiIcon type={datasetService.getType(type)!.meta.icon.type} />,
       });
     });
-
+    const recentlySelected: EuiSelectableOption[] = [
+      {
+        label: 'Recently Selected Data',
```

Comment:
Shouldn't this be `Recently selected data`

<img width="503" alt="Screenshot 2024-09-12 at 2 20 51 PM" src="https://github.com/user-attachments/assets/4aa621a0-6e9a-437f-b8c2-36351f75b05b">